### PR TITLE
🐛(models) add default language to user

### PR DESCRIPTION
**Pull request by sampaccoud**
## Purpose

The `createsuperuser` management command won't work if the language field is required and not set by default.

## Proposal

Add `LANGUAGE_CODE` as default language

### CODE REVIEW

File: tests/sandbox/test_management_commands.py [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+Test suite for magnify's main module
+"""
+from io import StringIO
+from unittest import mock
+
+from django.core.management import call_command
+from django.test import TestCase
+
+# sandbox
+from sandbox.settings import get_release
```
Comment (jbpenrath) [inline tests/sandbox/test_management_commands.py:11]: This is unused.
```suggestion
```

File: tests/sandbox/test_management_commands.py [hunk 2]
```diff
@@ -0,0 +1,31 @@
+"""
+Test suite for magnify's main module
+"""
+from io import StringIO
+from unittest import mock
```
Comment (jbpenrath) [inline tests/sandbox/test_management_commands.py:5]: This is unused.
```suggestion
```

File: src/magnify/apps/core/models.py [hunk 3]
```diff
@@ -76,6 +76,7 @@ class User(BaseModel, AbstractBaseUser, PermissionsMixin):
     language = models.CharField(
         max_length=10,
         choices=lazy(lambda: settings.LANGUAGES, tuple)(),
+        default=settings.LANGUAGE_CODE,
```
Comment (jbpenrath) [inline src/magnify/apps/core/models.py:79]: If nothing is yet in production, maybe you could recreate the initial migration?

Comment (sampaccoud) [reply src/magnify/apps/core/models.py:79]: True

**Review by jbpenrath — APPROVED** [review]
LGTM ✅

**Pull request merged**